examples/customresources/models/mynavigation/mynavigation.go

```diff
@@ -40,9 +41,10 @@ func newNav(ctx context.Context, deps registry.Dependencies, cfg config.Service,
 }
 
 type navSvc struct {
-	mu sync.RWMutex
-	loc *geo.Point
-	logger golog.Logger
+	generic.Unimplemented
```

Comment:
L26 of this file asserts that `navSvc` implements the `navigation.Service` interface that now includes `resource.Generic`.